lib/Botly.js

```diff
@@ -244,6 +261,7 @@ Botly.prototype.upload = function (options, callback) {
 };
 
 Botly.prototype.send = function (options, callback) {
+    let that = this;
```

Comment:
why do you need this line?
Arrow function keeps the scope and the `this` should be the correct one